packages/hoppscotch-ui/src/components/smart/Table.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="overflow-auto rounded-md border border-dividerDark shadow-md">
+    <table class="w-full">
+      <thead>
+        <slot name="head">
+          <tr
+            class="text-secondary border-b border-dividerDark text-sm text-left bg-primaryLight"
+          >
```

Comment:
It seems the current instances of usage have the `tr` element with the same set of classes. Should we move it out of the `slot`?

Reply:
In future, we might need to have custom classes for the tr. Hence the tr with the default classes is inside the slot with the scope that it can be overriden in the parent.